### PR TITLE
Added share button

### PR DESCRIPTION
**Pull request by Mi-Lan**
# Summary

Adds a share button.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "cross-env": "^7.0.3",
     "nanoid": "^4.0.0",
     "react-qrcode-logo": "^2.7.0",
+    "react-web-share": "^1.1.6",
```
Comment (adamazad) [inline package.json:11]: I thought we're trying keep this library lean :)

Comment (Mi-Lan) [reply package.json:11]: Screenshot 2022-09-02 at 23 58 47
Yeah I was thinking its just 6kb very minimal library

Comment (Mi-Lan) [reply package.json:11]: @adamazad Do you think we should do it from scratch?

Comment (adamazad) [reply package.json:11]: I think the modal design didn\t really go along with Nimi. @zamli?

**Comment by zamli** [comment]
I reviewed the issue and we could do some custom design for this to fit into our product.
image
https://www.figma.com/file/IhAXgag9wFFBLLCW7pkv6i/Nimi-%7C-Devconnect-AMS-2022?node-id=1774%3A38138

**Review by adamazad — APPROVED** [review]

**Pull request merged**